liquidity/liquidity.go

```diff
@@ -85,6 +91,12 @@ type Config struct {
 // Parameters is a set of parameters provided by the user which guide
 // how we assess liquidity.
 type Parameters struct {
+	// FailureBackOff is the amount of time that we require passes after a
+	// channel has been part of a failed loop out swap before we suggest
+	// using it again.
+	// TODO(carla): add exponential backoff
```

Comment:
Do you want to keep this comment here? There is so much to do still.

Reply:
It's my first TODO in the loop repo, so will leave it in. I do also think that un-swappable peers/channels will be important to address going forward with autoloop.

Reply:
There are three in this PR :stuck_out_tongue: